tests/contrib.print_proxy.mapfish_print/resources/expected_getspec_extract.json

```diff
@@ -35,6 +35,10 @@
       "Code": "ch.LaermempfindlichkeitsstufeninForce",
       "Text": "Lärmempfindlichkeitsstufen (in Nutzungszonen)"
     },
+    {
+      "Code": "ch.LaermempfindlichkeitsstufeninForce",
+      "Text": "Lärmempfindlichkeitsstufen (in Nutzungszonen)"
```

Comment:
maybe there should be the geometry type also appended here, so that there are no 2 identical entries, since this can be rather confusing.